backend/controllers/WebhookController.ts

```diff
@@ -72,9 +74,13 @@ export class WebhookController extends BaseController {
     }
   }
 
-  public postAnnouncementsWebhook(req: Request, res: Response) {
+  public async postAnnouncementsWebhook(req: Request, res: Response) {
```

Comment:
No need to mark this `async`, since it doesn't `await` anything

```suggestion
  public postAnnouncementsWebhook(req: Request, res: Response) {
```